### DISCUSSION TITLE
用 Tradytics 日交易 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#33] [Announcements]
# 用 Tradytics 日交易 - forecho's Blog

引言
Tradytics 是一个美股期权交易平台，提供了很多有用的数据，可以帮助我们日交易。今天就来介绍一下如何使用 Tradytics 日交易。
使用 Tradytics 日交易
主要看 TICKER DASHBOARDS 下面的三个 Dashboard，分别是：
Stocks Dashboard
主要目的就是添加 GEX 线

https://blog.forecho.com/tradytics-for-day-trading.html

**Comment by yeahwong** [comment]
请教一下，这个文章配图是什么软件截出来的？谢谢。

**Reply by forecho** [reply]
我使用的是 SetApp 里面的 Cleanshot X https://blog.forecho.com/use-setapp.html#cleanshot-x

同类型的软件也有不少。
